3_visualize/src/drought_learner_viz_components.R

```diff
@@ -885,5 +891,153 @@ frame_l <- function(p2_streamflow_learner_viz_df, p2_droughts_learner_viz_df,
   ggsave(out_png, width = 1200, height = 800, dpi = 300, units = "px")
 }
 
-
+frame_m <- function(p2_streamflow_learner_viz_df, p2_droughts_learner_viz_df,
+                    p2_1951_2020_drought_prop_site, p2_1951_2020_drought_prop_jd_7d,
+                    bottom_bars, canvas, inset, out_png){
+  
+  # Select correct station ID and threshold
+  p2_droughts_site <- p2_1951_2020_drought_prop_site %>%
```

Comment:
RC:  I'd avoid using this `targets` naming convention for variables that are defined within functions, as it may lead to confusion about which objects are and are not targets.

Reply:
Makes sense!